pinecone/client.go

```diff
@@ -436,22 +436,22 @@ type CreatePodIndexRequest struct {
 
 // ReplicaCount ensures the replica count of a pods-based Index is >1.
 // It returns a pointer to the number of replicas on a CreatePodIndexRequest object.
-func (req CreatePodIndexRequest) ReplicaCount() *int32 {
+func (req CreatePodIndexRequest) ReplicaCount() int32 {
 	x := minOne(req.Replicas)
-	return &x
+	return x
```

Comment:
is assignment and explicit return necessary?

Reply:
Nope, lemme clean this up - I ran through here too quick fixing the initial build.